src/main.rs

```diff
@@ -223,7 +197,13 @@ fn make_dir_if_not_exist(dir: &str) -> Result<(), io::Error> {
     Ok(())
 }
 /// Fuzz a given fuzz target
-fn run_target(target: String) -> Result<bool, Box<error::Error>> {
+fn exec_target<'a>(args: &ArgMatches<'a>) -> Result<(), Box<error::Error>> {
+    let target: String = args.value_of_os("TARGET").expect("TARGET is required").to_os_string()
+        .into_string().map_err(|_| "TARGET must be valid unicode")?;
+    let exec_args = args.values_of_os("ARGS");
+
+    let target_triple = "x86_64-unknown-linux-gnu";
```

Comment:
Hmm? `target` is the script to be fuzzed, not target triple. I also can see somebody naming their fuzz script a `x86_64-unknown-linux-gnu` and there’s no technical reason to forbid it?